gallery_dl/extractor/furaffinity.py

```diff
@@ -113,6 +113,9 @@ def _parse_post(self, post_id):
             data["gender"] = rh(extr('>Gender</strong>', '</div>'))
             data["width"] = pi(extr("<span>", "x"))
             data["height"] = pi(extr("", "p"))
+            data["folders"] = [rh(folder) for folder in extr(
+                '<h3>Listed in Folders</h3>',
+                '</section>').split('</a>') if rh(folder) != '']
```

Comment:
```suggestion
            data["folders"] = folders = []
            for folder in extr(
                    "<h3>Listed in Folders</h3>", "</section>").split("</a>"):
                folder = rh(folder)
                if folder:
                    folders.append(folder)
```

More code, but it calls `remove_html()` only once.
Walrus operator would be useful here when using a list comprehension, but that's Python 3.8+.